### ISSUE TITLE
[Feature request] Live location sharing

### ISSUE THREAD

**Issue by ralaud** [simplex-chat/simplex-chat#1663]
Hello, live location sharing would be a handy feature. If this can be implemented without sacrificing privacy.

**Comment by ralaud** [comment]
Thats interesting, this would also reduce battery consumption. 

I like your idea getting location per request.

**Comment by freemyminde** [comment]
> Those who have granted the sender permission to send their geographical location without intervention, will be notified in the chat with a copy of the data that have been sent to the applicant, including date and time, frequency of requests.
> 
> A kind of report on sensitive data sent that operates in full transparency. This avoids incorrect behaviour.

I agree with this. Best would be a map, where every point that is sent gets marked. So not every update takes more space in the chat. Otherwise the chat will quickly be spammed with these updates.

**Comment by menkaur** [comment]
I second this one, would be awesome to have

**Comment by shepitchakpavlo** [comment]
Voting +1

**Comment by saschadaemgen** [comment]
10/10
